docker/puppetdb/ssl-setup.sh

```diff
@@ -0,0 +1,454 @@
+#!/bin/sh
```

Comment:
This is a copy/paste/modification of the existing ssl-setup.erb (https://github.com/puppetlabs/puppetdb/blob/master/resources/ext/cli/ssl-setup.erb)

I had to make this file sh-compatible as there's no bash on alpine.

Most of this file is unchanged, other than:
* arrays replaced with space-separated strings
* EZBake calls for user/group replaced with ENV variables of the same value (puppetdb/puppetdb)
* interactive prompts for files/dirs are commented-out since it wasn't trivial to change (but totally doable)
* puppet config print calls replaced with hardcoded directory paths